content/docs/ops/runbook/rotating-secrets.md

```diff
@@ -0,0 +1,44 @@
+---
+menu:
+  docs:
+    parent: runbook
+
+title: Rotating Secrets
+---
+
+# Start by rotating access to various services and IAM accounts.
+
+- Access AWS
+- Rotate these specific IAM Users first
+    - `bosh_user`
+    - `concourse_user`
+    - `cf_user`
```

Comment:
We dropped `cf_user` in favor of instance profiles recently, and for some reason we've never actually provisioned the read-only admin user, so there aren't currently any credentials rotate there. Maybe it's time to either drop that user or add it to a stack--wdyt @LinuxBozo ?